data/shrix1.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Shriprasanna",
+  "type": "personal",
+  "bio": "Front-End developer based in India",
+  "avator": "https://github.com/shrix1.png",
```

Comment:
Hey, @shrix1 currently you don't need to add this `avatar` code. Please remove this line.
``` suggestion
 
```